### PR TITLE
Parametric batch submission at IC, toggle for LHE reading and always clean jets from photons

### PR DESCRIPTION
**Pull request by aelwood**
- Implemented parametric batch submission at IC. This is to circumvent the hard 2000 individual job limit. A file with all the job directories is saved and the jobs are submitted separately in a parametric way.
- The non lazy LHE extra information event reading was found to significantly increase the timing of the susyParameterScanAnalyzer (and GenAnalyzer) (increasing the fraction of time they took by ~10%) . Added a toggle to switch it off for the susyParameterScanAnalyzer). 
- Added an option to define the cleanJets as ones that are cleaned from photons as well.

### CODE REVIEW

File: PhysicsTools/HeppyCore/scripts/heppy_batch.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def batchScriptIC(jobDir):
 python {cmssw}/src/PhysicsTools/HeppyCore/python/framework/looper.py pycfg.py config.pck
 echo
 echo 'sending the job directory back'
-mv Loop/* ./
+cp -r Loop/* ./ && rm -r Loop
```
Comment (gpetruc) [inline PhysicsTools/HeppyCore/scripts/heppy_batch.py]: why this change?
usually move is cheaper than copy + delete (and doesn't use up twice the space on disk or afs while it is performed)

Comment (aelwood) [inline PhysicsTools/HeppyCore/scripts/heppy_batch.py]: The "Loop" directory still existed, which meant when resubmitting "Loop_1" would be made instead. I can change this to a mv then rm instead of a cp and rm.

**Comment by gpetruc** [comment]
Hi @aelwood , can you make the corresponding Heppy PR?

**Comment by aelwood** [comment]
@gpetruc done https://github.com/CERN-PH-CMG/cmg-cmssw/pull/359
